homework-submission/todo_react/src/App.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import "./App.css";
+import Info from "./Info";
+import data from "./data";
+import TodoLists from "./TodoLists";
+import ItemCreate from "./ItemCreate";
+
+
+class App extends Component {
+  state = {
+    todos: data
+  };
+
+  handleClick = (id) => {
+    const newTodos = this.state.todos;
```

Comment:
This is not a new state!
According to the `reference` vs `value` concepts, this is not a new copy of the state, But it's a reference to what is inside the state.

Which means what ever you will change in `newState`, is happening actually on the original `state`, but you can still see it on `newState` variable.